modules/tinymce/src/themes/silver/main/ts/ui/general/Button.ts

```diff
@@ -79,6 +79,21 @@ export const renderIconButton = (spec: IconButtonWrapper, action: (comp: AlloyCo
   return AlloyButton.sketch(iconButtonSpec);
 };
 
+const calculateClassesFromButtonType = (buttonType: 'primary' | 'secondary' | 'toolbar') => {
+  switch (buttonType) {
+    case 'primary':
+      return [ 'tox-button' ];
+      break;
```

Comment:
Is this some es lint rule requiring a break even if it has a return?

Reply:
This should not be needed. I'm guessing it was just missed in the refactoring as it was a mutable variable before.

Reply:
yeah I just search/replaced the `variable = ` to a `return`, oops